src/fides/api/service/connectors/base_connector.py

```diff
@@ -115,3 +115,20 @@ def dry_run_query(self, node: ExecutionNode) -> Optional[str]:
     @abstractmethod
     def close(self) -> None:
         """Close any held resources"""
+
+    def execute_standalone_retrieval_query(
+        self, node: ExecutionNode, fields: List[str], filters: Dict[str, List[Any]]
+    ) -> List[Row]:
```

Comment:
In order to be able to execute a connector query that doesn't check the nodes or incoming or outgoing edges, we define this new method to execute a "standalone" query. For now we're only implementing it for SQLConnector and Snowflake (client that requested this feature uses Snowflake and SQL connector seems to cover many cases).  This was to limit the scope of this PR